### PR TITLE
Post form from React

### PR DESCRIPTION
**Pull request by khannakshat7**
Resolves Issue #3 
### Description
The API url is `/api/leads/`. I have included name, email and message fields in the body of the request.

### How to run
In the frontend directory run following commands-

`npm install`
`npm start`

Try submitting the form and see request in the network tab in inspect element.

### Checklist
- [x] Code compiles correctly.
- [x] Changes are tested properly.
- [x] Added the README / documentation, if necessary.
- [x] Followed the Contributing guidelines.
- [x] Resolved merge conflicts, if any.

### CODE REVIEW

**Comment by khannakshat7** [comment]
@AnirudhAchal Please review and merge the PR

File: Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx [hunk 1]
```diff
@@ -18,7 +18,13 @@ class LeadFormContainerView extends Component {
     };
   }
 
-  handleSubmit() {}
+  handleSubmit() {
+    axios
+      .post("/api/leads/",this.state.lead)
```
Comment (AnirudhAchal) [inline Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx:23]: Could you use object destructuring?

File: Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx [hunk 2]
```diff
@@ -18,7 +18,13 @@ class LeadFormContainerView extends Component {
     };
   }
 
-  handleSubmit() {}
+  handleSubmit() {
```
Comment (AnirudhAchal) [inline Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx:21]: Could you format your code using an ESLinter?

**Review by AnirudhAchal — CHANGES_REQUESTED** [review]

File: Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx (hunk 2 above)
Comment (khannakshat7) [reply Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx:21]: Done please review

File: Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx (hunk 1 above)
Comment (khannakshat7) [reply Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx]: I think lead object is defined above so its fine. Or should I change it?

**Comment by khannakshat7** [comment]
@AnirudhAchal Please review and merge the PR

File: Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx (hunk 1 above)
Comment (AnirudhAchal) [reply Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx]: Take a look at some other functions in the react files. Can you store the lead object as `const { lead } = this.state;` for consistency?

Comment (khannakshat7) [reply Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx]: @AnirudhAchal Done Please review

**Comment by khannakshat7** [comment]
@AnirudhAchal Made few changes please review

**Review by AnirudhAchal — APPROVED** [review]
LGTM!

**Comment by khannakshat7** [comment]
@ikjot-2605 please add `Hacktoberfest` topic to the repo and `hacktoberfest-accepted` label to the repo. Otherwise, it won't be counted.

**Pull request merged**